netty/src/test/java/io/grpc/netty/ProtocolNegotiatorsTest.java

```diff
@@ -415,7 +415,9 @@ public void from_tls_clientAuthRequire_noClientCert() throws Exception {
         .trustManager(caCert)
         .build();
     Status status = expectFailedHandshake(channelCreds, serverCreds);
-    assertThat(status.getDescription()).isEqualTo("ssl exception");
+    if (!"ssl exception".equals(status.getDescription())) {
+      assertThat(status.getDescription()).startsWith("io exception");
```

Comment:
Do we know what's the full description message in this case?

Reply:
The description in one case (linux) is just "ssl exception" but in case of macos/Windows it starts with "io exception" and then includes some other context dependent text. I don't have a sample handy (and I could not retrieve the earlier runs/outputs

Reply:
Yes, if you can. `io exception` can happen for many reasons, and it'll be great if we could narrow it down to ssl-related errors (if possible).

Reply:
Done. Added a check to verify the desc contains "SslHandler"